### DISCUSSION TITLE
Install paddle paddle on CUDA 12.2

### DISCUSSION THREAD

**Discussion by philipnakhleh** [PaddlePaddle/PaddleOCR#12477] [Q&A]
Hello, I am trying to install paddle paddle on CUDA version 12.2 but I couldn't is there any way to run it?

**Comment by theolivenbaum** [comment]
Any updates on supporting CUDA 12.2?

**Comment by Koa28062004** [comment]
I also have problem when trying to download paddle-gpu for CUDA version 12.2. Are there any solutions ?
